src/pages/Contact/Contact.css

```diff
@@ -1,32 +1,18 @@
-@media all and (min-width: 480px) {
-    .background {
-        background-image: url('../../pics/adult-backlit-beach.jpg');
-        background-position: center;
-        background-repeat: no-repeat;
-        background-size: cover;
-        height: 100%;
-    }
-    .Contact {
-        margin: 0 auto;
-        max-width: 320px;
-        background-color: white;
-    }
-
-    .Contact form {
-        width: 100%;
-    }
+input[type=text] {
+    width: 100% !important;
+}
 
-    .Contact fieldset {
-        border: 1px solid grey;
-    }
+.contactForm {
+    display: flex;
+    justify-content: center;
+    padding-top: 30px;
+}
 
-    .Contact-item {
-        margin-top: 16px;
-    }
-    .Contact input,
-    textarea {
-        width: 100%;
-        padding: 0.5rem 0;
-        border: 1px solid grey;
-    }
+.header {
+    text-align: center;
+    padding-bottom: 1em; 
 }
+
+.header h2 {
+    color: #6831DE;
+}
```

Comment:
The code is not formatted properly.

Reply:
I don't see the problem with the format of this code block